container/apptabs.go

```diff
@@ -3,678 +3,312 @@ package container
 import (
 	"fyne.io/fyne/v2"
 	"fyne.io/fyne/v2/canvas"
-	"fyne.io/fyne/v2/driver/desktop"
-	"fyne.io/fyne/v2/internal"
 	"fyne.io/fyne/v2/layout"
 	"fyne.io/fyne/v2/theme"
 	"fyne.io/fyne/v2/widget"
 )
 
+// MaxAppTabs is the maximum number of visible tabs in an AppTab.
+// Any addition tabs are moved to the overflow menu.
+const MaxAppTabs = 7
+
+// Declare conformity with Widget interface.
+var _ fyne.Widget = (*AppTabs)(nil)
+
 // AppTabs container is used to split your application into various different areas identified by tabs.
 // The tabs contain text and/or an icon and allow the user to switch between the content specified in each TabItem.
 // Each item is represented by a button at the edge of the container.
 //
 // Since: 1.4
 type AppTabs struct {
-	widget.BaseWidget
```

Comment:
This change breaks `&AppTabs{Items: ...}` constructor - can we keep compatibility?

Reply:
Fixed